src/pkgcheck/checks/pkgdir.py

```diff
@@ -233,8 +272,23 @@ def feed(self, pkgset):
         if unknown:
             yield UnknownPkgDirEntry(sorted(unknown), pkg=pkg)
 
+        # we need to do Manifest separately, after we've got all the files
+        try:
+            manifest_stat = os.stat(pjoin(pkg_path, 'Manifest'))
```

Comment:
Can you please use `pathlib.Path` here - I plan to migrate from `pjoin` to `pathlib` for better readability and safety.